README.md

```diff
@@ -201,6 +201,22 @@ or if using dockerized Devnet:
 $ docker run -e RUST_LOG=<LEVEL> shardlabs/starknet-devnet-rs
 ```
 
+By default, logging of request and response data is turned off.
+To see the request or/and response body, additional level have to be provided via `RUST_LOG` environment variable.
+To log the request body use `REQUEST`, to log the response body use `RESPONSE`.
+
+NOTE! that logging request and response requires at least logging level `INFO`.
+
+Example:
+
+```
+$ RUST_LOG="REQUEST, RESPONSE" starknet-devnet
+```
+
+```
+$ RUST_LOG="REQUEST, RESPONSE, INFO" starknet-devnet
+```
```

Comment:
What does this mean? That the two commands are the same? If so, that could be stated just before the example lines.

Reply:
[ab5e11d](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/455/commits/ab5e11d659d7125a1927d0dfb2d2ed7838ab1512)